src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
It would be good if you can add more OOP to your Duke program to handle each process of the program separately, in their own respective class files.